homeassistant/components/freebox/button.py

```diff
@@ -39,6 +39,11 @@ class FreeboxButtonEntityDescription(
         device_class=ButtonDeviceClass.RESTART,
         async_press=lambda router: router.reboot(),
     ),
+    FreeboxButtonEntityDescription(
+        key="mark_calls_as_read",
+        name="Mark missed calls as read",
```

Comment:
I've name it `Mark missed calls as read` because it reminds the `Freebox missed calls` sensor, but indeed it mark all calls as read.

As other call types (`accepted` & `outgoing`) are already mark as read automatically, only `missed` call type are not read by default.

Reply:
It's also the only entity of the integration which does not contain the work "Freebox" in its name, but I think the name may be too long or odd otherwise.

What do you think ?

Reply:
I think that you should consider migration to `has_entity_name`

Reply:
I've removed "Freebox" to the Freebox restart button, other integrations don't, except ONVIF.

The entity_id changed from _button.reboot_freebox_ to _button.reboot_, adding to Breaking Changes

Reply:
The `entity_name` is different from the integration name.
In my case `entity_name` is "Freebox v8 (r1)", integration "Freebox".

Using `has_entity_name` for a good entity name.

Reply:
If there is a unique id, it is not a breaking change.

Reply:
There is

Reply:
Ups, but the unique id is base on the name ...

changing it to the key and making BC

Reply:
Done